Altis_Life.Altis/stringtable.xml

```diff
@@ -1041,7 +1041,7 @@
             <Italian>Il conto della gang non ha abbastanza denaro!</Italian>
             <Polish>Nie ma tego dużo na koncie bankowym gang!</Polish>
             <Russian></Russian>
-            <French>Vous n'avez pas autant dans votre gang!</French>
+            <French>Vous n'avez pas autant sur le compte de votre Gang !</French>
```

Comment:
Would a lower case `g` in `gang` be more appropriate considering it is not a proper noun? Same goes for all of the other capitalised words. I'm not trying to be annoying, but we're trying to hold a high and consistency standard on the repository.